.github/workflows/update-renovate-prs.yml

```diff
@@ -0,0 +1,46 @@
+name: Update renovate PRs
+
+on:
+  push:
+    branches:
+      - 'renovate/**'
+
+jobs:
+  check-metadata:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      id-token: write
+
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - uses: actions/setup-python@bd6b4b6205c4dbad673328db7b31b7fab9e241c0 # v4
+        with:
+          python-version: 3.7
+
+      - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4
+        with:
+          go-version: ^1
+
+      - name: Setup helm-docs
+        run: go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
+
+      - name: Run pre-commit
+        uses: pre-commit/action@646c83fcd040023954eafda54b4db0192ce70507 # renovate: tag=v3.0.0
+        continue-on-error: true
```

Comment:
Shouldn't this workflow fail if we can't run the pre-commit hooks here?

Reply:
Pre-commit always errors (exit code == 1) if if founds itself changing files. But in this case we want to use it TO change files. So we expect it to return exit code 1 and we want to continue the workflow afterwards. It's a `do_something() || true` kind of a hack.

Reply:
Ah I understand. Thank you for the explanation!